.travis.yml

```diff
@@ -20,11 +24,6 @@ before_script:
   - docker pull quay.io/keboola/syrup-cli:latest
 
 
-script:
-  # run test job inside KBC
-  # - docker run --rm -e KBC_STORAGE_TOKEN quay.io/keboola/syrup-cli:latest run-job $KBC_DEVELOPERPORTAL_APP $KBC_APP_TEST_CONFIG_ID test
```

Comment:
IMHO mat nejaky test v KBC nebolo zle.

Reply:
jj to by bylo fajn 👍  udělám až bude releasnutý v rámci https://keboola.atlassian.net/browse/COM-474